src/main/java/com/iota/iri/service/API.java

```diff
@@ -647,32 +859,49 @@ private synchronized AbstractResponse getTransactionsToApproveStatement(int dept
         return tips;
     }
 
+    /**
+     * <p>
+     *     Handles statistics on tip selection. 
+     *     Increases the tip selection by one use.
+     * </p> 
+     * <p>
+     *     If the {@link #getCounterGetTxToApprove()} is a power of 100, a log is send and counters are reset.
+     * </p>
+     */
     private void gatherStatisticsOnTipSelection() {
```

Comment:
@alon-e what do you mean?

Reply:
that private methods generally don't need docs. and above I mentioned it would be better to make these statistics methods private & not add javadocs to them - it makes them seem important :)

Reply:
Ah fair enough, I made them all private now, i think making them static is kind of halfway solution.
We need a statistics class for this instead. I will make a ticket!